app/helpers/application_helper.rb

```diff
@@ -47,13 +47,30 @@ def human_readable_date(date)
     Date.parse(date).strftime('%e %B %Y') # eg 3 August 2015
   end
 
+  def humanize_with_dashes(string)
+    string
+      .humanize
+      .tr('-', ' ')
+  end
+
+  def make_urls_wrappable(string)
+    # Assumes a full stop with a lowercase letter, number, or an underscore on
+    # both sides is part of a URL; then adds an invisible breaking space to
+    # allow wrapping.
+    # eg:
+    #  - Apples. Oranges. Pears. => Apples. Oranges. Pears.
+    #  - GOV.UK => GOV.UK
+    #  - registers.service.gov.uk => registers.&#8203;service.&#8203;gov.&#8203;uk
+    string
+      .gsub(/([a-z0-9_])\.([a-z0-9_])/, '\1.&#8203;\2')
+      .html_safe
```

Comment:
What's the thinking behind this approach rather than using CSS word-wrap?

Reply:
`overflow-wrap: break-word` is fine for a long word that goes outside of the table element - but I think that breaking on the `.` of the URL helps keep the table more readable. For example, with `break-word` you could end up with: 
```
registers.se
rvice.gov.uk
```
By adding an invisible space, you get:

```
registers.
service.
gov.uk
```